Amplitude/Amplitude.m

```diff
@@ -310,13 +319,13 @@ - (void)initializeApiKey:(NSString*) apiKey userId:(NSString*) userId
         return;
     }
 
-    (void) SAFE_ARC_RETAIN(apiKey);
+    SAFE_ARC_RETAIN(apiKey);
```

Comment:
I don't think the (void) does anything. Retain returns self, but it's not being saved to a variable. But I'm not entirely sure if it won't affect anything, so I will add the (void) back in